vulnerabilities/views.py

```diff
@@ -19,50 +19,118 @@
 from django.views import View
 from django.views.generic.edit import UpdateView
 from django.views.generic.list import ListView
+from packageurl import PackageURL
+from packaging import version
 
 from vulnerabilities import forms
 from vulnerabilities import models
+from vulnerabilities.forms import CVEForm
+from vulnerabilities.forms import PackageForm
+from vulnerablecode import settings
```

Comment:
https://docs.djangoproject.com/en/4.1/topics/settings/#using-settings-in-python-code

> Also note that your code should not import from either global_settings or your own settings file.

Use `from django.conf import settings` instead.

Reply:
Thanks @tdruez -- changed.